spring-cloud-stream-binder-rabbit-core/src/main/java/org/springframework/cloud/stream/binder/rabbit/properties/RabbitCommonProperties.java

```diff
@@ -231,6 +231,13 @@
 	 */
 	private boolean dlqSingleActiveConsumer;
 
+	/**
+	 * The bean name of a stream message converter to convert from a Spring AMQP Message
+	 * to a Stream Message.
+	 * @since 3.2
+	 */
+	private String streamStreamMessageConverterBeanName;
```

Comment:
Is this an intentional property name?

Reply:
Yes; there are two converters - Message<?> to AMQP Message, AMQP Message to Stream message.

Since they are both for stream, same prefix; second `Stream` to disambiguate.